Fix undeclared class issue on iOS

gentlejo opened the pull request.

Commit:
Fix undecleared class issue on iOS

faku99 commented:
First of all, thank you very much for taking the time to do a PR.

Could you please rename the changed files to STStereoPlugin.h and STStereoPlugin.m for consistency? Thank you in advance.

gentlejo commented:
@faku99 I met some error when build on iOS like below. So I've changed name. If I changed name I could fix the issue. Is there any other workaround?

Error output from Xcode build:
↳
** BUILD FAILED **
Xcode's output:
↳
=== BUILD TARGET Runner OF PROJECT Runner WITH CONFIGURATION Debug ===
/Users/gentlejo/Projects/myapp/ios/Runner/GeneratedPluginRegistrant.m:25:4: error: unknown receiver 'STStereoPlugin'; did you mean 'StereoPlugin'?
[STStereoPlugin registerWithRegistrar:[registry registrarForPlugin:@"STStereoPlugin"]];
^~~~~~~~~~~~~~
StereoPlugin
In module 'stereo' imported from /Users/gentlejo/Projects/myapp/ios/Runner/GeneratedPluginRegistrant.m:13:
/Users/gentlejo/Projects/myapp/build/ios/Debug-iphonesimulator/stereo/stereo.framework/Headers/StereoPlugin.h:5:12: note: 'StereoPlugin' declared here
@interface StereoPlugin : NSObject <FlutterPlugin>
^
1 error generated.

gentlejo commented:
@faku99 Have you check this?

faku99 commented:
I just did. I don't get the compilation error you're referring to in your PR message.

Did you update Flutter to the latest version? It might fix things up.

mglaman commented:
Would this fix #41?
